Specs/Number/Italian/NumberModel.json

```diff
@@ -1847,5 +1847,28 @@
         }
       }
     ]
+  },
+  {
+    "Input": "tr",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": []
+  },
+  {
+    "Input": "ha compiuto settant'anni",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "settant",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "70"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "ha compiuto settant anni",
```

Comment:
This is not a regular form, right? If the purpose here is to show this is not extracted, let's add a Comment attribute in the spec describing it.

Reply:
comment added